KuleumBridge/TabbarView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+struct TabbarView: View {
```

Comment:
```suggestion
struct TabBarView: View {
```

탭바는 Tab Bar 로 두 단어 입니다~